beps/bep_0003.rst

```diff
@@ -87,70 +92,115 @@ announce
 
 info
   This maps to a dictionary, with keys described below.
+  
+``piece layer``
+  A list of strings. Each string consists of concatenated hashes
+  of an intermediate merkle tree layer for each file. The layer is chosen so that
+  one hash represents one piece. For example if a piece size of 128KiB is used
+  then 3rd layer up from the leaf hashes is used.
```

Comment:
this paragraph implies that the leaf size is 16 kiB, perhaps that should be stated explicitly.